feat(catalog-backend-module-gitlab): add userTransformer and groupTransformer (#21903)
backstage/backstage: .changeset/neat-hotels-wink.md

---
'@backstage/plugin-catalog-backend-module-gitlab': patch
---

> add `groupTransformer`, `userTransformer` and `groupNameTransformer` to allow custom transformations when groups and users are created

Reviewer: Just a suggestion

```suggestion
Added the option to provide custom `groupTransformer`, `userTransformer` and `groupNameTransformer` to allow custom transformations of groups and users
```
